### PR TITLE
Removed SoftMax and added explanation

### PR DESCRIPTION
**Pull request by jvaleroliet**
SoftMax is not needed in Pytorch when using CrossEntropyLoss.

https://shaktiwadekar.medium.com/why-softmax-not-used-when-cross-entropy-loss-is-used-as-loss-function-during-neural-network-d77abd708715

https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html#crossentropyloss

### CODE REVIEW

**Comment by MLsound** [comment]
Hi @jvaleroliet! Thank you for this PR and for pointing out the redundancy. You are absolutely correct!

Please accept my apologies for the long delay in reviewing this; I truly appreciate your patience. I have approved the PR, and it will be merged in the upcoming maintenance release.

Thank you for being part of the community!

Best regards,
Alejandro

**Comment by jvaleroliet** [comment]
Hi Alejandro,
Thank you for the updates!
Regards

**Comment by MLsound** [comment]
Merged into [release/v2.0-update](https://github.com/musikalkemist/pytorchforaudio/tree/release/v2.0-update). We’ll be merging the full release branch into main once the remaining updates are finalized. Feel free to check out other open issues if you'd like to contribute further!
